app/controllers/photoTagController.js

```diff
@@ -1,7 +1,42 @@
+import { PhotoTagCreate, PhotoTagResponse } from "../models/apiModels/photoTag.js";
+import PhotoTagAccessor from "../databaseAccessors/photoTagAccessor.js";
+import { ErrorRepeatedTagName, ErrorUnexpected, HttpError } from "../error/errors.js";
+
+
 /**
  * PhotoTagController Class
  *
  * This class controls the behaviour of any web request
  * related to PhotoTags
  */
-export default class PhotoTagController {}
+export default class PhotoTagController {
+  /**
+   * Creates a PhotoTag with ref fields populated
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async create(req, res) {
+    try {
+      const body = new PhotoTagCreate(req.body);
```

Comment:
`body` is an interesting choice of variable name here. It might be unclear as to exactly what this value is when referred to later on in the code.